c-bindings/wrapper.h

```diff
@@ -0,0 +1,13 @@
+#include "picosha2.hpp"
+#include "../src/verifier.hpp"
```

Comment:
I would remove this include here and move it into `wrapper.cpp`. This interface doesn't depend on the verifier class itself.
Also, it would be best to not have a relative path here, it's something compilers don't always like (being able to "escape" include directories)